README.md

```diff
@@ -1,5 +1,111 @@
-# Coastal Ocean Assessment Tool
-This is a package for working on coastal assessment of [NEMO](https://www.nemo-ocean.eu/) data
+```
+__________________________________________________________________________________________
+
+
+                     ______    ___        _            _________ 
+                   .' ___  | .'   `.     / \          |  _   _  |
+                  / .'   \_|/  .-.  \   / _ \     .--.|_/ | | \_|
+                  | |       | |   | |  / ___ \   ( (`\]   | |    
+                  \ `.___.'\\  `-'  /_/ /   \ \_  `'.'.  _| |_   
+                   `.____ .' `.___.'|____| |____|[\__) )|_____|  
+                                              
+                          Coastal Ocean Assessment Toolbox
+          
+__________________________________________________________________________________________
+```
+
+** Notes on Object Structure and Loading (for contributors):
+
+COAsT is an object-orientated package, meaning that data is stored within Python object
+structures. In addition to data storage, these object contain methods which allow for 
+manipulation of this data.  An example of such an object is the NEMO object, which allows 
+for the storage and manipulation of NEMO output and domain data. It is important to
+understand how to load data using COAsT and the structure of the resulting objects.
+
+A NEMO object is created and initialised by passing it the paths of the domain and data 
+files. Ideally, the grid type should also be specified (T, U, V or F in the case of NEMO). 
+For example, to load in data from a file containing data on a NEMO T-grid:
+
+```
+import coast
+
+fn_data = '<path to T-grid data file(s)>'
+fn_domain = '<path to domain file>'
+
+data = coast.NEMO(fn_data, fn_domain, grid_ref='T')
```

Comment:
grid_ref="t-grid"

This is something I think we will have fixed in short order.